dawn/src/dawn/IIR/Extents.h

```diff
@@ -209,16 +209,16 @@ class HorizontalExtent {
   HorizontalExtent() = default;
 
   HorizontalExtent(ast::HorizontalOffset const& hOffset) {
```

Comment:
No change in this file. Was it properly formatted before?

Reply:
That may have been my enum PR. I don't think VSCode was configured correctly to auto-apply `clang-format` on save.